cmd/celestia/cmd_test.go

```diff
@@ -128,3 +128,23 @@ func TestBridge(t *testing.T) {
 			})
 	*/
 }
+
+func parseSignatureForHelpstring(methodSig reflect.StructField) string {
```

Comment:
let's not put a helper that's used in actual code in a test file

Reply:
we don't use it anywhere except current test